sound/soc/sof/sof-client-probes-ipc4.c

```diff
@@ -224,6 +248,41 @@ static int ipc4_probes_points_info(struct sof_client_dev *cdev,
 	return 0;
 }
 
+/**
+ * ipc4_probes_point_print - Human readable print of probe point descriptor
+ * @cdev:	SOF client device
+ * @buf:	Buffer to print to
+ * @size:	Available bytes in buffer
+ * @desc:	Describes the probe point to print
+ * @return:	Number of bytes printed or an error code (snprintf return value)
+ */
+static int ipc4_probes_point_print(struct sof_client_dev *cdev, char *buf, size_t size,
+				   struct sof_probe_point_desc *desc)
+{
+	struct device *dev = &cdev->auxdev.dev;
+	struct snd_sof_widget *swidget;
+	int ret;
+
+	swidget = sof_client_ipc4_find_swidget_by_id(cdev, SOF_IPC4_MOD_ID_GET(desc->buffer_id),
+						     SOF_IPC4_MOD_INSTANCE_GET(desc->buffer_id));
+	if (!swidget) {
+		dev_err(dev, "%s: Failed to find widget for module %lu.%lu\n",
+			__func__, SOF_IPC4_MOD_ID_GET(desc->buffer_id),
+			SOF_IPC4_MOD_INSTANCE_GET(desc->buffer_id));
+		ret = snprintf(buf, size, "%u,%u,%u\nUNKNOWN buffer_id 0x%08x\n",
```

Comment:
can we keep this in one line ?
```c
"%u,%u,%u\tUNKNOWN buffer_id 0x%08x\n"
```
and you are printing the `desc->buffer_id` twice?